tests/server/index.js

```diff
@@ -1,24 +1,125 @@
+const mongoose = require('mongoose');
 const tape = require('tape');
 const supertest = require('supertest');
 const exphbs = require('express-handlebars');
+require('env2')('./config.env');
 
+mongoose.connect(process.env.DB_URL, {
+  useMongoClient: true
+});
+
+const { mockCollection } = require('../../database/startup_schema.js');
 const server = require('../../src/server.js');
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', () => {
+  mockCollection.remove({startupName: 'FAC'}, (err) => {
+    if (err) {
+      console.log(err);
+      return
+    }
+    console.log('removed');
+  });
+  console.log("connected db");
+})
+
 
 tape('Test home route', (t) => {
-  console.log(exphbs.create().render('src/views/search_home.hbs'));
-  var html1 = '<h1>test test</h1>\n';
+  let html = '<h1 class="near-white mv0 f1">Arab innovators</h1>\n';
   supertest(server).get('/').end((err, res) => {
     t.error(err, 'No Error');
     t.equal(res.status, 200, 'Should equal 200');
-    t.equal(res.text, html1, 'html is the same')
+    t.equal(res.text.includes(html), true, 'html is the same')
     t.end();
   });
 });
 
-tape('Test submission form route', (t) => {
-  supertest(server).get('/submissionForm').end((err, res) => {
+tape('Test profile page route', (t) => {
+  let html = '<h2 class="f1 absolute pt2">MyCity</h2>\n';
+  let html2 = '<h1> Sorry we couldn\'t find any startup in our database that match your selections</h1>\n'
+  supertest(server).get('/startupProfile/595cbfe08d75e77913c05c0e').end((err, res) => {
     t.error(err, 'No Error');
     t.equal(res.status, 200, 'Should equal 200');
+    t.equal(res.text.includes(html), true, 'The right startup was found');
+  })
+  supertest(server).get('/startupProfile/sdgadfasd').end((err, res) => {
+    t.equal(res.text.includes(html2), true, 'Recieved 404 page');
     t.end();
-  });
-});
+  })
+})
+
+
+tape('Test for register startup route', (t) => {
+  let expected = {
+    'startup-name': 'FAC',
+    'founder-name': 'Dan',
+    'coFounder-name': '',
+    'description': 'Coding bootcamp for everyone',
+    Select: 'Software',
+    'funding-stage': 'Seed-Funded',
+    'startup-email': 'mario91sss@gmail.com',
+    'startup-website': 'foundersandcoders.com'
+  }
+  supertest(server).post('/registerStartup')
```

Comment:
if supertest managed to post the url and end the request then check if theres no error thats a whole test by itself and istanbul confirmed that it is covered